### PR TITLE
Datacollection of volume mesh from mesh directory

### PR DESCRIPTION
**Pull request by rassaire**
I found that creating a data collection of volume meshes from a directory could only read vtk files while Scalismo can read up to three formats (vtk, vtu and inp). I have added other formats (inp and vtu)

### CODE REVIEW

**Comment by marcelluethi** [comment]
Thanks for adding this. I changed the target branch to release-0.18, as we never directly merge anything into master.

Your PR has failed because the formatting is not correct. Please run 
```sbt scalafmtAll``` 
and commit the corrected formatting. 

You can instruct intelliJ to reformat your code correctly when saving. In this way you will never run into this problem again. See this [link](https://scalameta.org/scalafmt/docs/installation.html).

File: src/main/scala/scalismo/statisticalmodel/experimental/dataset/DataCollectionOfVolumeMesh.scala [hunk 1]
```diff
@@ -110,7 +110,7 @@ object DataCollectionOfVolumeMesh {
     val meshFileNames = meshDirectory
       .listFiles()
       .toSeq
-      .filter(fn => fn.getAbsolutePath.endsWith(".vtk") || fn.getAbsolutePath.endsWith(".stl"))
+      .filter(fn => fn.getAbsolutePath.endsWith(".vtk") || fn.getAbsolutePath.endsWith(".vtu")||fn.getAbsolutePath.endsWith(".inp"))
     val (meshes, ioErrors) = DataUtils.partitionSuccAndFailedTries(for (meshFn <- meshFileNames) yield {
       MeshIO.readMesh(meshFn).map(m => TetrahedralMesh3D(m.pointSet, referenceMesh.tetrahedralization))
```
Comment (kahrpatrick) [inline src/main/scala/scalismo/statisticalmodel/experimental/dataset/DataCollectionOfVolumeMesh.scala]: I just noticed that calling `DataCollectionOfVolumeMesh.fromMeshDirectory` leads to an empty data collection as the meshes are loaded with the wrong IO function.

```suggestion
      MeshIO.readTetrahedralMesh(meshFn).map(m => TetrahedralMesh3D(m.pointSet, referenceMesh.tetrahedralization))
```

**Comment by Andreas-Forster** [comment]
I removed myself as a reviewer as there are already two active on this PR.

**Comment by marcelluethi** [comment]
@rassaire I fixed the issue with the PR, such that we can close it and move on. Unfortunately I don't seem to have permissions to contribute to the PR. Can you give me the rights to edit this PR?

**Comment by rassaire** [comment]
Hi Marcel, I am wondering why you don't have rights to edit.

**Comment by rassaire** [comment]
@marcelluethi  I have this from my side which should allow you to edit. 
github

**Comment by rassaire** [comment]
> Thanks for adding this. I changed the target branch to release-0.18, as we never directly merge anything into master.
> 
> Your PR has failed because the formatting is not correct. Please run
> `sbt scalafmtAll`
> and commit the corrected formatting.
> 
> You can instruct intelliJ to reformat your code correctly when saving. In this way you will never run into this problem again. See this [link](https://scalameta.org/scalafmt/docs/installation.html).

Thanks, @marcelluethi . Sorry I didn't see your comment earlier. I'll check that out.

**Pull request merged**